### PR TITLE
Gate map

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
The new map is as experimental a format as possible. 

All separated into separate asteroids connected by a giant metal ring. Travel between departments is done through a network of portal gates, or through shuttles, which almost every department has.

## Why / Balance
Six months ago I promised Emisse to port this map to Wizden. I'm making good on that promise

## Technical details
Given that the map is outdated, a little bit scratched from migration from another build, as well as not checked for new standards, DO NOT MERGE
This PR will be open long enough to test, polish, and finalize all the necessary things.
Ed TODO:
- [x] Remove warp points, add nav beacons
- [x] More Cameras for AI
- [x] shitty Gate links
  - [ ] #32134 Gimmick Gate mechanic 
- [x] Station anchor
- [ ] A nuclear warhead doesn't have the power to destroy THIS THING
- [ ] Wait for #31653
- [x] Delete chromite island
  - [x] Move AI to Science island
  - [x] Move telecomms and camera servers to Engi
  - [x] Move Perma to Sec island
- [ ] #32099 for Medical
- [ ] a LOT of bugs from playtests

Emisse TODO:
- [ ] TEG engi access

## Media
Gate-0-min

## Requirements

- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added new map Gate

### CODE REVIEW

**Comment by TheShuEd** [comment]
This, huh, is a real high-pop map.

image

File: Resources/Prototypes/Maps/Pools/default.yml [hunk 1]
```diff
@@ -13,3 +13,4 @@
   - Packed
   - Reach
   #- Train
+  - Gate
```
Comment (UbaserB) [inline Resources/Prototypes/Maps/Pools/default.yml:16]: its in alphabetical order

**Comment by Deserty0** [comment]
Glory to all mappers

**Comment by Ilya246** [comment]
every dept is the size of reach

**Comment by Djungelskog2** [comment]
oh hey i had a map idea exactly like this, cool to see something come of the same concept

**Comment by NotSoDana** [comment]
Oh no, this undead

**Comment by ArcaneOcto** [comment]
This looks AWESOME. Is the whole station powered by a 3 core AME + solars? When this is finished I think it could easily be my favorite map.

**Comment by VitusVeit** [comment]
This is really cool, especially the departments' shuttles.

**Comment by TheShuEd** [comment]
> > This looks AWESOME. Is the whole station powered by a 3 core AME + solars? When this is finished I think it could easily be my favorite map.
> 
> I see a bunch of RTGs here and there powering each department.
> 
> I like how this map is basically 7 departments in a trenchcoat pretending to be a whole station. The scaffolding linking them cannot be destroyed, can it ?

image

**Comment by EthanQix** [comment]
> > them cannot be destroyed, can it ?
> > image

I mean if they can, this station is ridiculously vulnerable to explosive sabotage and meteor storms xD

**Comment by TheShuEd** [comment]
that sounds familiar... Hmm. oh, right, the train!!

**Comment by vladkaron14R** [comment]
> Поезд превращается в уроборус

Well, I don't know, Trends are very popular at night on Russian servers + it is suitable for small online so we are waiting for at least some kind of processing of the card that will appear soon

**Comment by Awlod** [comment]
Yes-Yes-Yes-Yes!!!!!!

**Comment by Plykiya** [comment]
this is train but good

**Comment by Awlod** [comment]
> this is train but good because even if it gets segmented, the gateways still work

The train left the map rather later than the gate

**Comment by KingFroozy** [comment]
Oh, I remember this, got stuck with a bunch of people in sec during zombie outbreak once, good map though!

**Comment by WhiteGorila** [comment]
Cool in concept but this looks like a pain in the ass to navigate. Ie see train

**Comment by TheShuEd** [comment]
Gate-0-min

Updated

**Comment by superjj18** [comment]
> I mean if they can, this station is ridiculously vulnerable to explosive sabotage and meteor storms xD

I feel entire sections of this station can detached in like less than a minute by someone with an rcd

**Comment by TheShuEd** [comment]
Current map status: Awaiting a full-fledged review. This is a HUGE map that has already spent months of work, and for sure Emisse will find what global needs to be redone here. Until they do, I don't really want to polish the map in the current layout to perfection, as it might be a waste of effort.

**Comment by SlimmSlamm** [comment]
I can't wait until this is merged. This will actually be the best map.

**Comment by TheShuEd** [comment]
image

**Comment by TheShuEd** [comment]
image

**Comment by TheShuEd** [comment]
Gate-0-min

**Comment by lzk228** [comment]
In the name of the father, the son and the self merge. Amen.

**Comment by Cojoke-dot** [comment]
there ain't no meteor shielding on that singulo setup, it's going to loose to a meteor so quickly...

**Comment by SlimmSlamm** [comment]
> there ain't no meteor shielding on that singulo setup, it's going to loose to a meteor so quickly...

On the bright side the station is so big that you probably won't have to worry about the singulo going anywhere else besides Engineering.

**Pull request merged**